.github/workflows/swift.yml

```diff
@@ -44,7 +44,7 @@ jobs:
   # without waiting for the macOS build
   build-and-test-macos:
     if: github.repository_owner == 'github'
-    runs-on: macos-12-xl
+    runs-on: macos-13-xlarge
```

Comment:
This WF I'm not sure whether it should be on 13 or 14, but in the bigger sense it doesn't really matter. Let's keep this as-is.